osbs/build/build_request.py

```diff
@@ -166,6 +166,8 @@ def dj(self):
     def has_ist_trigger(self):
         """Return True if this BuildConfig has ImageStreamTag trigger."""
         triggers = self.template['spec'].get('triggers', [])
+        if not triggers:
```

Comment:
Or maybe better yet to set it to an empty list when triggers variable is defined:
```python
triggers = self.template['spec'].get('triggers') or []
```

In either case, this is fine as is to me.